### ISSUE TITLE
Include and Bad Request

### ISSUE THREAD

**Issue by brenoroosevelt** [FriendsOfCake/crud-json-api#50]
According to the JSON API specification about ```include``` query parameter (http://jsonapi.org/format/#fetching-includes) : 

> If an endpoint does not support the include parameter, it MUST respond with 400 Bad Request to any requests that include it.

Consider this and the follow:
```php
public function view()
{
  $this->Crud
    ->listener('jsonApi')
    ->config('queryParameters.include.blacklist', true);
    // OR  ->config('queryParameters.include.whitelist', false);
  return $this->Crud->execute();
}
```
The request below should respond with 400 Bad Request : 
```
GET /posts/1?include=author
```
i.e., if blacklist is ```true```, or whitelist is ```false```, the endpoint does not support any include and should responds with 400 Bad Request, but it is responding with: 200 OK. Indeed, the plugin responds with Bad Request  only if  the resource is unable to identify a relationship (in the case of "include").

**Comment by bravo-kernel** [comment]
Fix in #51, released as https://github.com/FriendsOfCake/crud-json-api/releases/tag/0.2.5

**Comment by bravo-kernel** [comment]
Much much appreciated @brenoroosevelt !!
